### ISSUE TITLE
Is there any any to set trainable weights in loss function

### ISSUE THREAD

**Issue by busyyang** [keras-team/keras#13844]
I have set a custom loss function as below:
~~~
def cross_entropy_with_penalty(y_true, y_pre):
    penalty = K.constant([1, 5, 5])
    return -tf.reduce_sum((y_true * penalty) * K.log(y_pre + 1e-7)) - tf.reduce_sum(
        ((1 - y_true) * penalty) * K.log(1 - y_pre + 1e-7))
~~~
and it works. However I hope the `penalty ` can be trained, is there any way for it?

**Comment by Saduf2019** [comment]
@busyyang 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
